config.ini.default

```diff
@@ -1,5 +1,4 @@
-#!/bin/sh
```

Comment:
I'm pretty sure this doesn't need the she-bang. It worked without it

Reply:
🤷